app/models/repo.js

```diff
@@ -131,17 +131,31 @@ Repo.reopen({
   }.property(),
 
   branches: function() {
-    var builds;
-    builds = this.store.query('build', {
+    var branches;
+    branches = this.store.query('branch', {
       repository_id: this.get('id'),
-      branches: true
+      sort_by: 'default_branch,name'
     });
-    builds.then(function() {
-      return builds.set('isLoaded', true);
+
+    branches.then(function() {
+      return branches.set('isLoaded', true);
+    });
+    return branches;
+  }.property(),
+
+  crons: function() {
+    var id = this.get('id');
+    return this.store.filter('cron', {
+      repository_id: id
+    }, function(cron) {
+      return cron.get('branch.repoId') === id;
     });
-    return builds;
   }.property(),
 
+  sortedCrons: Ember.computed.sort('crons', function(a, b) {
+    return a.get('branch.name') > b.get('branch.name');
+  }),
```

Comment:
Unfortunately this currently only work after e.g. adding a cron job. We think it could be because `branch` will be loaded async but we did not figure out if or how we could use a `then` inside the sort method.